mindmeld/components/dialogue.py

```diff
@@ -689,6 +692,194 @@ def _get_dialogue_handler(self, dialogue_state):
         return handler
 
 
+class AutoEntityFilling():
+    """A special dialogue flow sublcass to implement Automatic Entitiy (Slot) Filling
+    (AEF) that allows developers to prompt users for completing the missing
+    requirements for entity slots.
+
+    Attributes:
+        app (Application): The application that initializes this flow.        
+    """
+
+    logger = mod_logger.getChild("AutoEntityFilling")
+    """Class logger."""
+
+    def __init__(self, entrance_handler, entity_form, app, **kwargs):
+        self.app = app
```

Comment:
we should make all of these into private variables
```suggestion
        self._app = app
```

Reply:
I was keeping this behavior consistent with dialogueflow class, can make it a hidden variable if that's ideal